.github/workflows/nightlies.yml

```diff
@@ -0,0 +1,179 @@
+on:
+  schedule:
+    - cron: '0 2 * * *'
```

Comment:
This runs every night, but only creates a release if there is a change on the repo, correct?

Reply:
yep, that's correct!